install.sh

```diff
@@ -1019,22 +1019,22 @@ ohai "Downloading and installing Homebrew..."
     ) || exit 1
   fi
 
+  if [[ -n "${ADD_PATHS_D-}" ]]
+  then
+    execute_sudo "${MKDIR[@]}" /etc/paths.d
+    echo "${HOMEBREW_PREFIX}/bin" | execute_sudo tee /etc/paths.d/homebrew
+    execute_sudo "${CHOWN[@]}" root:wheel /etc/paths.d/homebrew
+    execute_sudo "${CHMOD[@]}" "a+r" /etc/paths.d/homebrew
+  elif [[ ":${PATH}:" != *":${HOMEBREW_PREFIX}/bin:"* ]]
+  then
+    warn "${HOMEBREW_PREFIX}/bin is not in your PATH.
+    Instructions on how to configure your shell for Homebrew
+    can be found in the 'Next steps' section below."
```

Comment:
Actually, it might be worth saving this warning for after the `brew update` below, since that can be noisy. Mind moving this warning later?

Reply:
OK, I'll work it into another PR.